src/profile/v100/entity/account/account.rs

```diff
@@ -1,5 +1,30 @@
 use crate::prelude::*;
 
+pub trait IsEntity {
+    /// An order set of `EntityFlag`s used to describe certain Off-ledger
+    /// user state about Accounts or Personas, such as if an entity is
+    /// marked as hidden or not.
+    fn flags(&self) -> EntityFlags;
```

Comment:
we can let this trait grow... it is rather big in Swift world: https://github.com/radixdlt/sargon/blob/profile_integration_stage_1_cyon_batch_14/apple/Sources/Sargon/Protocols/EntityProtocol.swift#L11

We cannot share Rust trait to Swift :/ since UniFFI puts `: AnyObject` restriction on all `trait` we export into Swift Protocols....